src/tests/countList.js

```diff
@@ -0,0 +1,3 @@
+const countRecipes = (recipe) => recipe.length;
```

Comment:
- [ ] Hi,  The purpose of the test is to test the actual functions that your production code uses currently you are testing a function that has been created in the tests folder. You need to test the function that you are actually using in the modules folder that your website actually uses. Please kindly fix this issue.

Reply:
changed the path, create a separate module for the counter and import it into Test and listItem files.